internal/logger/logger.go

```diff
@@ -15,34 +15,60 @@
 package logger
 
 import (
+	"fmt"
 	"io"
 	"log"
+	"log/syslog"
 	"os"
 
 	"github.com/jacobsa/daemonize"
 )
 
+// ProgrammeName constant is used while writing the logs to syslog file, it is
+// used while filtering the gcsfuse log-message from the syslog file, since
+// syslog file contains all the system related logs from other programmes too.
```

Comment:
rephrase it.

Syslog file contains logs from all different programmes running on the VM. Programme name is prefixed to all logs written to syslog. This constant is used to filter the logs from syslog and write it to respective log files(gcsfuse.log for gcsfuse.)